src/templates/base.html

```diff
@@ -130,8 +130,10 @@
         {% block top_div_extra_content %}{% endblock %}
     </div>
     <div class="ui container grid">
-        {% block content %}
-        {% endblock %}
+        <div class="row">
```

Comment:
Ah, great catch, we do need row here so it doesn't futz up margins?

Reply:
Correct, it keeps margins working properly